bim2sim/sim_settings.py

```diff
@@ -733,6 +733,12 @@ def __init__(self):
                     'boundary information regarding external/internal.',
         for_frontend=True
     )
+    create_plots = BooleanSetting(
```

Comment:
This could be added to base sim settings if we have plots for other plugins.

Reply:
let's keep it there for now. We might then rename this one here to `create_beps_plots' or something similar and add other settings to the other plugins.